src/ibeo_core.cpp

```diff
@@ -946,6 +992,66 @@ std::vector<Point3DL> ScanData2208::get_scan_points()
   return v;
 }
 
+ScanData2209::ScanData2209() :
+  IbeoTxMessage(true, false, false)
+{}
+
+void ScanData2209::parse(const std::vector<uint8_t>& in)
+{
+  ibeo_header.parse(in);
+
+  uint16_t hdr = IBEO_HEADER_SIZE;
+
+  scan_start_time = read_be<NTPTime>(in, hdr);
+  scan_end_time_offset = read_be<uint32_t>(in, hdr + 8);
+
+  uint32_t flags = read_be<uint32_t>(in, hdr + 12);
+  fused_scan = ((flags & 0x00000100) > 0);
+  mirror_side = ((flags & 0x00000200) > 0) ? REAR : FRONT;
+  coordinate_system = ((flags & 0x00000400) > 0) ? VEHICLE : SCANNER;
+
+  scan_number = read_be<uint16_t>(in, hdr + 16);
+  scan_points = read_be<uint32_t>(in, hdr + 18);
+  number_of_scanner_infos = read_be<uint8_t>(in, hdr + 22);
+
+  // printf("%d SCAN POINTS REPORTED\n", scan_points);
```

Comment:
Remove commented code.

Reply:
Hello together,

is this going to happen some time? We have the same problem and are interested in this PR.

Best regards